basis/models.py

```diff
@@ -35,9 +35,12 @@ class PersistentModel(models.Model):
     class Meta:
         abstract = True
 
-    def delete(self, *args, **kwargs):
-        self.deleted = True
-        self.save(*args, **kwargs)
+    def delete(self, using=None, force=False):
```

Comment:
>Why did you introduce using?

It is in [django's own delete function](https://docs.djangoproject.com/en/1.7/ref/models/instances/#django.db.models.Model.delete) so it needs to be passed into super. I didn't introduce it, it is just more specific than `**kwargs`.

> Can you please write a test when using is used?

Sure, but it is used to select database in case you have several databases in `settings.DATABASES`. Is it worth it to add and sync several databases in the test suite?